Cargo.toml

```diff
@@ -10,4 +10,6 @@ license = "Apache-2.0 OR BSD-3-Clause"
 edition = "2018"
 
 [dependencies]
-libc = ">=0.2.98"
+libc = ">=0.2.39"
+serde = { version = ">=1.0.27", features = ["derive"] }
```

Comment:
This matches also version 2.0.0 or larger, I think you'd want to specify it as ^1.0.27, which is equivalent to >=1.0.27 <2.0.0, so at least the major version wouldn't change. Alternatively, you could specify it as ~1.0.27 or >=1.0.27 <1.1.0

Reply:
good call! will do

Reply:
done